eslint.config.js

```diff
@@ -30,16 +26,7 @@ const config = [
             "**/*.mts",
             "**/*.tsx",
         ],
-
         languageOptions: {
-            ecmaVersion: "latest",
```

Comment:
[`ecmaVersion` defaults to `"latest"`](https://eslint.org/docs/latest/use/configure/language-options#specifying-javascript-options).